### PR TITLE
Basic home page (without map and info about supermarkets)

### PR DESCRIPTION
**Pull request by MarchenkoVladislav**
image
The first button in navbar is in pushed condition and others no.

### CODE REVIEW

File: src/main/kotlin/com/discounts/nearby/controller/HomeController.kt [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import org.springframework.web.bind.annotation.GetMapping
+
+/**
+ * @author Created by Vladislav Marchenko on 19.11.2020
+ */
+@Controller
+class HomeController {
+
+    @GetMapping("/")
+    fun homePage() : String = "/homePage"
```
Comment (SHvatov) [inline src/main/kotlin/com/discounts/nearby/controller/HomeController.kt:13]: In order to maintain the readability of the code and also single common style, please, when you are committing changes in IDEA, check this options as well:
Screenshot from 2020-11-21 11-27-49

**Review by SHvatov — COMMENTED** [review]
Page looks pretty normal to me, but maybe we should increase the padding between the buttons on the nav bar?

**Comment by SHvatov** [comment]
And  also, lets make the font size of the text in the nav bar bigger, so that it differs from regular text

**Review by SHvatov — CHANGES_REQUESTED** [review]
Everything else is ok, so after these small fixes I will merge this request right away

**Comment by SHvatov** [comment]
[Add basic template of main page and controller for returning this page](https://trello.com/c/Pb76f4Al/20-add-basic-template-of-main-page-and-controller-for-returning-this-page)

**Comment by MarchenkoVladislav** [comment]
New version of home page
image

**Review by SHvatov — APPROVED** [review]

**Pull request merged**